src/mmds/src/lib.rs

```diff
@@ -221,7 +218,7 @@ mod tests {
         // Test Ok path.
         let request = b"GET http://169.254.169.254/ HTTP/1.0\r\n\r\n";
         let mut expected_response = Response::new(Version::Http10, StatusCode::OK);
-        let body = "age\nname/\nphones/".to_string();
+        let body = r#"{"age":"43","name":{"first":"John","second":"Doe"},"phones":{"home":{"RO":"+40 1234567","UK":"+44 1234567"},"mobile":"+44 2345678"}}"#.to_string();
```

Comment:
Please use appropriate formatting here. Using `r#` is powerful because it allows you to use spaces and tabs to format the jsons.